kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/orders/ordering/OrderData.java

```diff
@@ -0,0 +1,10 @@
+package com.kodilla.good.patterns.orders.ordering;
+
+import com.kodilla.good.patterns.orders.data.*;
+
+public interface OrderData {
```

Comment:
"An interface in the Java programming language is an abstract type that is used to specify a behaviour that classes must implement"
In this case Order or OrderRequest represents POJO (Plain old Java object) which stores only data, and do not have any specific behaviour. So creating interface is bad choice in my opinion.